wasmbinding/bindings/msg.go

```diff
@@ -223,3 +226,25 @@ type ResubmitFailure struct {
 type ResubmitFailureResponse struct {
 	FailureId uint64 `json:"failure_id"`
 }
+
+type Dex struct {
+	Deposit                  *types.MsgDeposit                  `json:"deposit"`
+	Withdrawal               *types.MsgWithdrawal               `json:"withdrawal"`
+	PlaceLimitOrder          *MsgPlaceLimitOrder                `json:"place_limit_order"`
```

Comment:
I don't really like the copy+paste from proto file with subtle changes made: on proto updates, we'll have to manually reflect the changes in such copy+pasted structures. what do you think about making kind of wrapper structs for each struct where we replace `time.Time` with `uint64` and implementing a simple custom `UnmarshalJSON` method for them. rough example (compiles but not sure that works):

```golang
type QueryEstimatePlaceLimitOrderRequest dextypes.QueryEstimatePlaceLimitOrderRequest

func (r *QueryEstimatePlaceLimitOrderRequest) MarshalJSON() ([]byte, error) {
	type Alias dextypes.QueryEstimatePlaceLimitOrderRequest
	return json.Marshal(&struct {
		ExpirationTime int64 `json:"expiration_time"`
		*Alias
	}{
		ExpirationTime: r.ExpirationTime.Unix(),
		Alias:          (*Alias)(r),
	})
}

func (r *QueryEstimatePlaceLimitOrderRequest) UnmarshalJSON(data []byte) error {
	type Alias dextypes.QueryEstimatePlaceLimitOrderRequest
	aux := &struct {
		ExpirationTime int64 `json:"expiration_time"`
		*Alias
	}{
		Alias: (*Alias)(r),
	}
	if err := json.Unmarshal(data, &aux); err != nil {
		return err
	}
	expirationTime := time.Unix(aux.ExpirationTime, 0)
	r.ExpirationTime = &expirationTime
	return nil
}
```

The point is if we ever change the proto e.g. add/remove fields, change some fields type, etc, we won't have to reflect the changes here in the copy+pasted structure. it will work under the hood because of the `Alias` auxiliary type.

custom unmarshal JSON guide: https://choly.ca/post/go-json-marshalling/